resolwe_bio/processes/variant_calling/gatk_merge_vcfs.py

```diff
@@ -0,0 +1,105 @@
+"""Run GATK MergeVcfs."""
+from plumbum import TEE
+
+from resolwe.process import (
+    Cmd,
+    DataField,
+    FileField,
+    ListField,
+    Process,
+    SchedulingClass,
+    StringField,
+)
+
+
+class GatkMergeVcfs(Process):
+    """Combine multiple variant files into a single variant file using GATK MergeVcfs."""
+
+    slug = "gatk-merge-vcfs"
+    name = "GATK MergeVcfs"
+    category = "GATK"
+    process_type = "data:variants:vcf:mergevcfs"
+    version = "1.0.0"
+    scheduling_class = SchedulingClass.BATCH
+    requirements = {
+        "expression-engine": "jinja",
+        "executor": {
+            "docker": {"image": "public.ecr.aws/genialis/resolwebio/dnaseq:6.1.0"}
+        },
+        "resources": {
+            "cores": 1,
+            "memory": 16384,
+            "storage": 200,
+        },
+    }
+    data_name = "Combined variants"
+
+    class Input:
+        """Input fields for GatkMergeVcfs."""
+
+        vcfs = ListField(DataField("variants:vcf"), label="Input data (VCFs)")
+        ref_seq = DataField(
+            "seq:nucleotide",
+            label="Reference sequence",
+            required=False,
+            description="Optionally use a sequence dictionary file (.dict) "
+            "if the input VCF does not contain a complete contig list.",
+        )
+
+    class Output:
+        """Output fields for GatkMergeVcfs."""
+
+        vcf = FileField(label="Merged VCF")
+        tbi = FileField(label="Tabix index")
+        species = StringField(label="Species")
+        build = StringField(label="Build")
+
+    def run(self, inputs, outputs):
+        """Run analysis."""
+
+        combined_variants = "combined_variants.vcf.gz"
+        combined_variants_index = combined_variants + ".tbi"
+        vcf_list = "input_variant_files.list"
+        species = inputs.vcfs[0].output.species
+        build = inputs.vcfs[0].output.build
+
+        if not all(vcf.output.species == species for vcf in inputs.vcfs):
+            self.error(
+                "Species information must be the same for all of the input VCF files."
+            )
+
+        if not all(vcf.output.build == build for vcf in inputs.vcfs):
+            self.error(
+                "Genome build information must be the same for all of the input VCF files."
+            )
+
+        with open(vcf_list, "w") as input_vcfs:
+            for vcf in inputs.vcfs:
+                input_vcfs.write(f"{vcf.output.vcf.path}\n")
+
+        args = [
+            f"I={vcf_list}",
+            f"O={combined_variants}",
+        ]
```

Comment:
We should also support setting the garbage collection options and java heap size in advanced options. 
`--java-options -XX:ParallelGCThreads= and -Xmx`
Both can greatly impact performance and memory usage.